src/core/main.cpp

```diff
@@ -223,7 +224,17 @@ void fileCheck( QString &file )
 	}
 }
 
+int usageError(const QString& message)
+{
+	qCritical().noquote() << QString("\n%1.\n\nTry \"%2 --help\" for more information.\n\n")
```

Comment:
I think you should not append a '.' to the `message` (`%1.`) here, otherwise the error messages for most of these messages will have two periods at the end of the error message. You will have to add a period to `noInputFileError` though.

Reply:
Removed '.' from the rest of the messages via daee35ebe86bf91c4c4ba40b1600e5f927ae1d08.